### PR TITLE
CN-5624, CN-5627, CN-5630: Improve loading to WalletLanding

### PR DESCRIPTION
**Pull request by djalfirevic**
## Checklist (Mandatory):

- [x] I have named the PR correctly (Check PR1)
- [x] Check the commit’s or even all commits message styles matches our requested structure. (issue number, etc.) (Check CR10)
- [x] I have tested new code locally (Check CR3)
- [x] I have done a smoke test of the larger feature locally (Check CR3)
- [x] I have added at least two reviewers (Check PR5)
- [x] I have assigned the PR to myself (Check PR4)
- [x] I have merged develop (parent) branch into my feature branch and resolved all conflicts (Check PR7)

## Checklist (Optional):

- [ ] I have added necessary documentation, including instructions how to test (Check CR8)
- [ ] I have added tests that prove my fix/implementation is effective or that my feature works
- [ ] I have written the code according to the adopted style guide? (Check CR7)

## Additional notes

> What does this implement/fix? Explain your changes. (optional)
> Are there any scripts that need running (eg. yarn install)?
> Are there any edge cases that need checking?
> Are there any specific test instructions?

[MVPWorkshop Pull Request & Code Review Guidelines](https://docs.google.com/document/d/1TcuU1JBhlGZLtpy0osSNinXQ2piP_yrDuRcXdqaOS_U/edit#heading=h.5b6kpdr4btq8)

### CODE REVIEW

File: app/components/screens/Home/Home.js [hunk 1]
```diff
@@ -54,11 +57,23 @@ class Home extends Component {
...
     if (!appInitialized) {
-      await appUtil.initializeThirdPartyServices();
-      await actions.initCelsiusApp();
+      appUtil.initializeThirdPartyServices();
+      actions.getInitialCelsiusData();
+
+      getSecureStoreKey(SECURITY_STORAGE_AUTH_KEY).then(token => {
+        actions.getProfileInfo();
+      });
+      2;
```
Comment (fjovakaric) [inline app/components/screens/Home/Home.js:69]: 2? :)

File: app/components/screens/Home/Home.js [hunk 2]
```diff
@@ -54,11 +57,23 @@ class Home extends Component {
...
-    const { actions, appInitialized } = this.props;
+  componentDidMount() {
+    const { actions, appInitialized, user } = this.props;
     if (!appInitialized) {
-      await appUtil.initializeThirdPartyServices();
-      await actions.initCelsiusApp();
+      appUtil.initializeThirdPartyServices();
+      actions.getInitialCelsiusData();
+
+      getSecureStoreKey(SECURITY_STORAGE_AUTH_KEY).then(token => {
```
Comment (fjovakaric) [inline app/components/screens/Home/Home.js:66]: why then, why not await?

File: app/components/screens/Home/Home.js [hunk 3]
```diff
@@ -54,11 +57,23 @@ class Home extends Component {
...
+        actions.getProfileInfo();
+      });
+      2;
+      actions.getUserAppSettings();
+
+      if (user.has_pin) {
```
Comment (fjovakaric) [inline app/components/screens/Home/Home.js:72]: Do we have the user here?

File: app/components/screens/VerifyProfile/VerifyProfile.js [hunk 4]
```diff
@@ -298,12 +298,7 @@ class VerifyProfile extends Component {
     const style = VerifyProfileStyle();
 
     return (
-      <RegularLayout
-        padding="0 0 0 0"
-        fabType={
-          hideBack || appState.match(/inactive|background/) ? "hide" : "main"
-        }
-      >
+      <RegularLayout padding="0 0 0 0" fabType={"hide"}>
```
Comment (fjovakaric) [inline app/components/screens/VerifyProfile/VerifyProfile.js:301]: this will always hide FAB

File: app/constants/UI.js [hunk 5]
```diff
@@ -262,7 +262,6 @@ const CEL_PAY_TYPES = {
 };
 
 const INITIAL_API_CALLS = [
```
Comment (fjovakaric) [inline app/constants/UI.js:264]: Do we need this now?

File: app/components/screens/Home/Home.js (hunk 2 above)
Comment (djalfirevic) [reply app/components/screens/Home/Home.js:66]: Will switch it to `await` in order to be consistent.

File: app/constants/UI.js (hunk 5 above)
Comment (djalfirevic) [reply app/constants/UI.js:264]: Removed.

**Pull request merged**